### PR TITLE
Fix Azure Custom Script Field Extra Arrays

### PR DESCRIPTION
**Pull request by nzthiago**
When going back to edit an existing Cluster in a Deploy stage of a pipeline, the JSON created for the Azure custom script field (fileUris) would keep adding arrays [] around it if it was already set. This prevents the extra array from being created again.
@scotmoor @rguthriemsft @duftler PTAL

### CODE REVIEW

**Comment by scotmoor** [comment]
@anotherchrisberry PTAL as well. Will need a rebase.

File: app/scripts/modules/azure/serverGroup/serverGroup.transformer.js [hunk 1]
```diff
@@ -80,7 +80,14 @@ module.exports = angular.module('spinnaker.azure.serverGroup.transformer', [
 
       if (typeof command.customScriptsSettings !== 'undefined') {
         configuration.customScriptsSettings.commandToExecute = command.customScriptsSettings.commandToExecute;
-        configuration.customScriptsSettings.fileUris = [command.customScriptsSettings.fileUris];
+        if (Array.isArray(command.customScriptsSettings.fileUris))
+        {
```
Comment (anotherchrisberry) [inline app/scripts/modules/azure/serverGroup/serverGroup.transformer.js]: opening bracket should be on the same line, i.e.
```javascript
if (Array.isArray(command.customScriptsSettings.fileUris)) {
```

File: app/scripts/modules/azure/serverGroup/serverGroup.transformer.js [hunk 2]
```diff
@@ -80,7 +80,14 @@ module.exports = angular.module('spinnaker.azure.serverGroup.transformer', [
...
+          configuration.customScriptsSettings.fileUris = command.customScriptsSettings.fileUris;
+        }
+        else
+        {
```
Comment (anotherchrisberry) [inline app/scripts/modules/azure/serverGroup/serverGroup.transformer.js]: Move to line above, i.e.
```javascript
else {
```

**Comment by anotherchrisberry** [comment]
Very minor formatting issues; LGTM otherwise

**Comment by anotherchrisberry** [comment]
ping @nzthiago - just need to address the formatting issues and rebase and we can get this merged

**Comment by nzthiago** [comment]
@anotherchrisberry sorry for the delay! Rebased and updated the formatting issues

**Comment by rguthriemsft** [comment]
@nzthiago  Will need to squash into a single commit.

**Comment by jrsquared** [comment]
@nzthiago If you get this rebased and squashed we can get it in...

**Comment by nzthiago** [comment]
I created a new pull request

**Pull request closed without merging**